charts/opentelemetry-operator/release/release-checklist.md

```diff
@@ -0,0 +1,29 @@
+# Prerequisites
+
+- [ ] Make sure you have installed KUTTL which will be used to do the smoke tests later. See [KUTTL website](https://kuttl.dev/docs/)
```

Comment:
We have some smoke tests in the operator using kuttl as well. Would it make sense to reuse one of them? We create a CR and ensure that the deployment exists and is in ready state:

Install step
https://github.com/open-telemetry/opentelemetry-operator/blob/902b5f60b93e5f8cce99ea67a6c8f8716c4ca65d/tests/e2e/smoke-simplest/00-install.yaml#L1-L24

Assert:
https://github.com/open-telemetry/opentelemetry-operator/blob/902b5f60b93e5f8cce99ea67a6c8f8716c4ca65d/tests/e2e/smoke-simplest/00-assert.yaml#L1-L6

Reply:
Yes, actually we use the KUTTL smoke tests from the Operator repository (please see [here](https://github.com/open-telemetry/opentelemetry-helm-charts/pull/52/files#diff-71db0aaa38245101ee6e7ded84d75067181ede554fd22f3df22b2a442899bf0fR21)). The `kuttl-test.yaml` will run all the e2e tests under `tests/e2e` directory.